indico/modules/events/abstracts/util.py

```diff
@@ -146,14 +148,15 @@ def create_mock_abstract(event):
                         contribution=contribution,
                         primary_authors=[englert, brout],
                         secondary_authors=[guralnik, hagen, kibble, higgs],
-                        locator={'confId': -314, 'abstract_id': 1234},
+                        locator={'confId': -314, 'abstract_id': 1234, 'uuid': None},
```

Comment:
why? the locator never contains the UUID

Reply:
Forgot to remove it..